### PR TITLE
Revert "meh"

### PR DESCRIPTION
**Pull request by DerZade**
Reverts gruppe-adler/gruppe_adler_mod#206

@AdlerSalbei Your PR (#206) included commits from 2018. I just wanted to make sure that everything was right. If this is not the case, please merge this PR and try again :D

### CODE REVIEW

**Comment by AdlerSalbei** [comment]
I'm confused, why would you revert me bringing a branch up to date. So that I can work on it with out having issues in the future?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,11 +11,11 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Build with HEMTT
-      uses: gruppe-adler/action-release-with-hemtt@2.2.0
+      uses: gruppe-adler/action-release-with-hemtt@2.0.0
```
Comment (AdlerSalbei) [inline .github/workflows/ci.yml:14]: Why the lower version here?

**Comment by DerZade** [comment]
> I'm confused, why would you revert me bringing a branch up to date. So that I can work on it with out having issues in the future?

I'm not reverting. I'm suggesting to revert your changes. That how you use a PR 😛. It's not just merging changes, it's requesting to merge changes.

I just found it confusing that your PR contained various commits from different times (some from 2018 and some from 2022), sometimes with no correlation between commits at all. That didn't seem right to me at all.
You didn't give anyone a chance to challenge your changes in your PR, because you merged it instantly. So I opened this PR and challenge them here 😇

**Comment by DerZade** [comment]
Okay apparently I'm completely stupid. You merged master into your own branch. I assumed the other way around 🤦‍♂️ 
Maybe opening GitHub at 2AM is not the best idea.

---

But why the fuck would you open a PR for merging master into your WIP branch?

```
git checkout applyACEMedicalDamageViaZeus
git merge master
```
or even
```
git checkout applyACEMedicalDamageViaZeus
git rebase master
```

gif

**Comment by AdlerSalbei** [comment]
I don't see the issue. With out having to open an Terminal. You can easily move the master over :P

**Pull request closed without merging**